Misc/NEWS.d/next/Library/2021-07-26-10-46-49.bpo-44493.xp3CRH.rst

```diff
@@ -0,0 +1 @@
+Add missing terminated NUL in sockaddr_un's length
```

Comment:
Perhaps some words like "This was potentially observable when using non-abstract AF_UNIX datagram sockets to processes written in another programming language." would add context for the reader as to when it might have happened?

Reply:
Ok.